tests/expected/c/00911-pascal_ptr.c

```diff
@@ -9,4 +9,5 @@ char* main()
 }
 
 void foo(int* a, int* b, int* c)
-{}
+{
```

Comment:
We can probably amend the config used for this test to avoid this unneeded change